build-setup/linux.md

```diff
@@ -389,32 +389,43 @@ Begin by cloning the IPEX repository:
 git clone https://github.com/intel/intel-extension-for-pytorch.git
 cd intel-extension-for-pytorch
 git fetch --all
-git checkout v1.13.100+cpu
+git checkout v2.1.0+cpu
 git submodule sync
 git submodule update --init --recursive
 git config --global --add safe.directory `pwd`
 ```
 
-IPEX expects that PyTorch build directory contains a file `build-version`, which contains the PyTorch version string. This file is not created by the PyTorch build process, so we need to create it manually:
+IPEX expects that PyTorch build directory contains a file `build-hash`, which contains the PyTorch git revision. This file is not created by the PyTorch build process, so we need to create it manually:
 ```bash
- echo "1.13.1+cpu\n" > ${PYTORCH_SRC_DIR}/torch/build-version
+ (cd ${PYTORCH_SRC_DIR}/torch/ && git rev-parse HEAD > build-hash)
 ```
 
 This assumes that you have cloned PyTorch in the directory `${PYTORCH_SRC_DIR}` in the step above. Make sure that this path is correct.
 
-Building IPEX requires a lot of memory. To reduce this requirement, we can patch the IPEX build system to lower the number of parallel processes. We use `sed` to replace the call to `multiprocessing.cpu_count()` with a constant `1` in the file `setup.py`:
+Building IPEX requires a lot of memory. To reduce this requirement, we can set the MAX_JOBS environment variable to lower the number of parallel processes:
 ```bash
-sed -i 's/multiprocessing.cpu_count()/1/g' setup.py
+export MAX_JOBS=1
+```
+
+IPEX expects that the `blas-devel` library package be installed:
+```bash
+yum install blas-devel.x86_64
```

Comment:
This needs some due diligence. The library that gets installed here is `libblas.so.3`. Does IPEX end up requiring that at runtime? If so then that's a problem as we cannot guarantee that it will be present on end user systems.

Or if IPEX doesn't need to link `libblas` at runtime, why is it needed at build time? Are there environment variables that can be set to say not to use this?